streams/src/test/java/org/apache/kafka/streams/StreamsBuilderTest.java

```diff
@@ -58,13 +55,6 @@
     private final StreamsBuilder builder = new StreamsBuilder();
     private final Properties props = StreamsTestUtils.topologyTestConfig(Serdes.String(), Serdes.String());
 
-    @Test(expected = TopologyException.class)
-    public void testFrom() {
```

Comment:
Why do we remove this test? Rename -> `shouldNotAllowToResueAutoGeneratedProcessorName` ?

Reply:
It is just a renaming to make its name consistent with others, I will rename it with non-capitalized name.

Reply:
EDIT: actually for this test, I removed it because it was covered in `TopologyTest#shouldNotAllowToAddSourcesWithSameName`.